pkg/services/ngalert/remote/remote_secondary_forked_alertmanager.go

```diff
@@ -160,9 +171,26 @@ func (fam *RemoteSecondaryForkedAlertmanager) CleanUp() {
 }
 
 func (fam *RemoteSecondaryForkedAlertmanager) StopAndWait() {
+	// Stop the internal Alertmanager.
 	fam.internal.StopAndWait()
+	// Stop our alert senders.
 	fam.remote.StopAndWait()
-	// TODO: send config and state on shutdown.
+
+	// Send config and state to the remote Alertmanager.
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
I don't think we should timeout here -- basically we want to allow a Grafana pod as much as it needs to ensure the state is saved (we have a global timeout of 30s anyways)